### DISCUSSION TITLE
iOS Widget

### DISCUSSION THREAD

**Discussion by deadchimis** [LeoKlaus/plappa#53] [Feature Requests]
Any plans to make a Widget to manipulate the app without the need to open it fully?
For example on audible's main app, or say something like Prologue:
![IMG_0127](https://github.com/LeoKlaus/plappa/assets/26066201/104f9e72-67a4-496f-80db-9780782032fc)

**Comment by LeoKlaus** [comment]
I really like that idea!

**Comment by cwhite616** [comment]
I have a “stack” of widgets which include YouTube player, Libby player, pocket casts. I’d love to add plappa to the mix, so all of my audio is in one stack, but it needs a widget to do that! I swipe up and down on this widget to cycle through my audio apps.

![IMG_6774](https://github.com/user-attachments/assets/96b37f17-829b-41df-a8ea-0391559f0cf2)

**Comment by seannymurrs** [comment]
I agree that a home screen widget would be a great addition to Plappa. Any update on whether or not this is planned in the near future?

**Comment by ScottOfford** [comment]
I know this is an older thread, but I've been using this as a stopgap until an official widget can be created.

It's a shortcut set to "Resume last played audiobook or episode" added to my stack

![plappa_shortcut](https://github.com/user-attachments/assets/0b1e0718-5b35-46b7-8bb2-d4f69629abf8)
